### ISSUE TITLE
Tooling for managing topics

### ISSUE THREAD

**Issue by JonasJurczok** [zalando/saiki-buku#28]
Hi,

it would be awesome to have some integrated tooling to view the status and create the list of topics (as everyone (Fowler etc.) suggest to have all topics and their respective configuration in version control).

For this purpose I created https://github.com/zalando/onboarding-buku-config which currently only works for our specific usecase. But maybe you can use it as a baseline for your ideas.

Basically it just gives you a local shellscript that can either issue a ./kafka-topics.sh --describe or a ./kafka-topics.sh --create on one of your kafka nodes.

There are some issues with the general usability at the moment (see issues of the project) but again maybe it serves as idea.

**Comment by chutium** [comment]
@drummerwolli sounds interesting for us? @mcehlert similar to what you have done for pemetaan or?

**Comment by JonasJurczok** [comment]
Is there any news on this?

**Comment by drummerwolli** [comment]
i haven't had the time yet to implement the version control part in https://github.com/zalando/saiki-pemetaan yet :-(
